src/hotspot/share/cds/filemap.cpp

```diff
@@ -213,7 +213,7 @@ void FileMapHeader::populate(FileMapInfo *info, size_t core_region_alignment,
   _compressed_class_ptrs = UseCompressedClassPointers;
   _max_heap_size = MaxHeapSize;
   _use_optimized_module_handling = MetaspaceShared::use_optimized_module_handling();
-  _use_full_module_graph = MetaspaceShared::use_full_module_graph();
+  _has_full_module_graph = CDSConfig::is_dumping_full_module_graph();
```

Comment:
`_uses_full_module_graph` would seem more appropriate than `use` or `has`

Reply:
The name is consistent with other fields in filemap.hpp:

```
  bool _has_non_jar_in_classpath;     // non-jar file entry exists in classpath
  bool _has_platform_or_app_classes;  // Archive contains app classes
  bool _has_full_module_graph;        // Does this CDS archive contain the full archived module graph?
```

Reply:
Okay makes more sense when expressed like that - but it is not obvious that the MetaspaceShared API should be read as applying "to this CDS archive".